docs/proposals/VIP_TEMPLATE.md

```diff
@@ -0,0 +1,135 @@
+---
+layout: default
+title: Venice Improvement Plan
+parent: Developer Guides
+permalink: /docs/proposals/VIP_TEMPLATE.md
```

Comment:
Would probably be good to have a landing page for proposals which presents a table of contents of all VIPs, either auto-generated from the child pages, if that's possible, or manually curated (which may be better if we want to display extra info, e.g. the status, release version, or other details...

We could leave that for later since we don't have any VIPs to present yet but anyway...

cc @nisargthakkar in case you have opinions/advice on page structure.

Reply:
Yeah. Planning to do a landing page. Let's get the first one in hopefully @ZacAttack might soon publish something after this gets merged. 

Let me do a landing page anyway in anticipation that this will land soon.